web/themes/custom/genlyd_aarhus/genlyd_aarhus.theme

```diff
@@ -113,6 +113,14 @@ function genlyd_aarhus_preprocess_node__activity(&$variables) {
     $imageUrl = ImageStyle::load('activity')->buildUrl($image);
   }
 
+  // Attach facebook share button js and drupalSettings.
+  $variables['#attached']['library'][] = 'itk_activity/fb_share';
+  $variables['#attached']['drupalSettings']['fbShare'] = [
```

Comment:
Please prefix variables in settings with module name:

variables['#attached']['drupalSettings']['genlyd_aarhus']['fbShare']

to ensure that i never will clash with other modules etc.